ui/hooks/sources.ts

```diff
@@ -33,25 +34,33 @@ export function useListSources(
           .helmRepositories;
         const buckets = (bucketsRes as ListBucketsResponse).buckets;
         const charts = (chartRes as ListHelmChartsResponse).helmCharts;
-
-        return [
-          ..._.map(repos, (r) => ({
-            ...r,
-            kind: FluxObjectKind.KindGitRepository,
-          })),
-          ..._.map(hrs, (c) => ({
-            ...c,
-            kind: FluxObjectKind.KindHelmRepository,
-          })),
-          ..._.map(buckets, (b) => ({
-            ...b,
-            kind: FluxObjectKind.KindBucket,
-          })),
-          ..._.map(charts, (ch) => ({
-            ...ch,
-            kind: FluxObjectKind.KindHelmChart,
-          })),
+        const ErrorList: ListError[] = [
+          ...repoRes.errors,
+          ...helmReleases.errors,
+          ...bucketsRes.errors,
+          ...chartRes.errors,
         ];
+        return {
+          result: [
+            ..._.map(repos, (r) => ({
+              ...r,
+              kind: FluxObjectKind.KindGitRepository,
+            })),
+            ..._.map(hrs, (c) => ({
+              ...c,
+              kind: FluxObjectKind.KindHelmRepository,
+            })),
+            ..._.map(buckets, (b) => ({
+              ...b,
+              kind: FluxObjectKind.KindBucket,
+            })),
+            ..._.map(charts, (ch) => ({
+              ...ch,
+              kind: FluxObjectKind.KindHelmChart,
+            })),
+          ],
+          errors: [..._.uniqBy(ErrorList, "clusterName")],
```

Comment:
Maybe do the uniqBy in the rendering component too just so we're not losing info at this point. (number of errors etc).

Reply:
Then we can also pretend clusterName doesn't exist as an attribute too. Maybe the core error renderer doesn't even look at the `clusterName` key and just print out the `message` attr.